### PR TITLE
TRACING-4063: document cumulativetodeltaprocessor

### PR DESCRIPTION
**Pull request by andreasgerstmayr**
Version(s):
all supported

Issue:
https://issues.redhat.com/browse/TRACING-4063

Link to docs preview:
https://73746--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-configuration-of-otel-collector#cumulativetodelta-processor_otel-configuration-of-otel-collector

QE review:
- [ ] QE has approved this change.

Additional information:

@max-cx

### CODE REVIEW

File: modules/otel-collector-components.adoc [hunk 1]
```diff
@@ -557,7 +557,36 @@ config: |
...
-You can optionally create an `attribute_source` configuratiion, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+You can optionally create an `attribute_source` configuration, which defines where to look for the attribute in `from_attribute`. The allowed value is `context` to search the context, which includes the HTTP headers, or `resource` to search the resource attributes.
+
+[id="cumulativetodelta-processor_{context}"]
+=== Cumulative to Delta processor
+
+The Cumulative to Delta processor is currently a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] feature only.
+
+The Cumulative to Delta processor converts monotonic, cumulative sum and histogram metrics to monotonic, delta metrics.
+Non-monotonic sums and exponential histograms are excluded.
```
Comment (max-cx) [inline modules/otel-collector-components.adoc:568]: @andreasgerstmayr, does this sentence mean that

A) The processor does not convert non-monolithic sums and exponential histograms
OR
B) The processor excludes non-monolithic sums and exponential histograms when it performs the conversion
?

Comment (andreasgerstmayr) [reply modules/otel-collector-components.adoc:568]: A

File: modules/otel-collector-components.adoc [hunk 2]
```diff
@@ -557,7 +557,36 @@ config: |
...
+
+.OpenTelemetry Collector custom resource with an enabled Cumulative to Delta processor
+[source,yaml]
+----
+config: |
+  processors:
+    cumulativetodelta:
+      include: <1>
+        match_type: strict <2>
```
Comment (max-cx) [inline modules/otel-collector-components.adoc:577]: @andreasgerstmayr, since `<1>` and `<3>` are optional, should this be optional too?

**Comment by max-cx** [comment]
@andreasgerstmayr, you are welcome to close this PR because I opened https://github.com/openshift/openshift-docs/pull/75128.

**Comment by andreasgerstmayr** [comment]
Closed in favor of #75128.

**Pull request closed without merging**